cpp_utils/src/cpp/logging/LogConfiguration.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file LogConfiguration.cpp
+ *
+ */
+
+#include <cpp_utils/logging/LogConfiguration.hpp>
+#include <cpp_utils/utils.hpp>
+
+namespace eprosima {
+namespace utils {
+
+
+LogConfiguration::LogConfiguration()
+    : verbosity{VerbosityKind::Warning, FuzzyLevelValues::fuzzy_level_default}
+{
+    LogFilter filter_template;
+    filter_template[VerbosityKind::Info] = "";
+    filter_template[VerbosityKind::Warning] = "";
+    filter_template[VerbosityKind::Error] = "";
+    filter.set_value(filter_template, FuzzyLevelValues::fuzzy_level_default);
+}
+
+bool LogConfiguration::is_valid(
+        Formatter& error_msg) const noexcept
+{
+    // Verifica si verbosity es uno de los valores permitidos
+    if (verbosity.get_value() != VerbosityKind::Error &&
```

Comment:
Is this necessary? If `verbosity` is of type `VerbosityKind` and `VerbosityKind` is an enum with `Error`, `Warning`, and `Info`, then this condition will always be true.